delira/models/backends/tf_eager/abstract_network.py

```diff
@@ -0,0 +1,155 @@
+import abc
+import typing
+import tensorflow as tf
+import numpy as np
+from delira.models.abstract_network import AbstractNetwork
+
+
+class AbstractTfEagerNetwork(AbstractNetwork, tf.keras.layers.Layer):
```

Comment:
maybe inherit from [tf.Module](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/Module) in future. Though I might actually prefer tf.keras.layers.Layer, as it provides nice additions/extensions such as trainable flag

Reply:
Added this as reminder to #165